spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -131,25 +132,25 @@ def is_defined(self, objtxt, force_import=False):
         """Return True if object is defined"""
         try:
             return self.call_kernel(
-                interrupt=True, blocking=True
```

Comment:
Will these methods work while on debugging? I mean, I thought it was necessary to interrupt the kernel to make these calls while debugging.

Reply:
I changed it so that blocking implies interrupt